src/Form/AbstractChoiceField.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MyParcelCom\JsonSchema\FormBuilder\Form;
+
+use InvalidArgumentException;
+use MyParcelCom\JsonSchema\FormBuilder\Properties\SchemaProperty;
+use MyParcelCom\JsonSchema\FormBuilder\Properties\SchemaPropertyType;
+use MyParcelCom\JsonSchema\FormBuilder\Translations\LabelTranslationCollection;
+
+abstract class AbstractChoiceField implements FormElement
```

Comment:
I personally do not add "Abstract" or "Interface" or "Enum" of elements that are defined with these keywords already. If you think about it - it does not really make a difference if a class is abstract, enum or interface - when a code uses these classes they provide the same features - you call methods or access constants etc.

```suggestion
abstract class ChoiceField implements FormElement
```